README.md

```diff
@@ -21,15 +21,21 @@ This image is available on Docker Hub: [lequal/sonarqube](https://hub.docker.com
 
 Since inception, this image has been designed to be used in production. Thus, leaving the default admin password (namely "admin") will never be an option. To this extent, a new password for the admin account must be given by setting the environment variable `SONARQUBE_ADMIN_PASSWORD`.
 
-To run the image locally
+To run the image locally:
+
 ```sh
 # Recommended options
-docker run --name lequalsonarqube \
-        --rm \
-        --stop-timeout 1 \
-        -p 9000:9000 \
-        -e SONARQUBE_ADMIN_PASSWORD="admin password of your choice" \
-        lequal/sonarqube:latest
+$ docker run --name lequalsonarqube \
+             --rm \
+             --stop-timeout 1 \
```

Comment:
Why?

Reply:
In Docker-CAT it was useful because docker could not kill the entrypoint of the container. However, it is useless in our image right now.